### PR TITLE
BaSP-TG-73: Super admin redux implementation

### PR DESCRIPTION
**Pull request by IvanHGo**
Worked on this branch: @IvanHGo @Agustin-Chazarreta
In the first commit you can see we started the problematic implementing the redux setup and creating the methods following the weekly class.
Later we notice that the problematic needed diferents states for each action so I added them in the following commits.
When the methods GET, PUT, POST and DELETE were finished I corriged some issues and moved them to the thunks file for better funcionality. 
For more information or errors, let me know.

### CODE REVIEW

File: src/Components/SuperAdmins/index.jsx [hunk 1]
```diff
@@ -40,19 +47,20 @@ function SuperAdmins() {
...
-        });
-    } catch (error) {
-      console.error(error);
-    }
+    dispatch(getSuperAdmins());
   }, []);
 
+  const getById = (ids) => {
+    setIsOpenEdit(true);
+    fetch(`${process.env.REACT_APP_API_URL}/super-admins/${ids}`)
```
Comment (cellamarematias) [inline src/Components/SuperAdmins/index.jsx:55]: Hi! You must remove this fetch from the index to the thunks.

File: src/Components/SuperAdmins/index.jsx [hunk 2]
```diff
@@ -15,17 +20,19 @@ function SuperAdmins() {
     email: '',
     password: ''
   };
+
   // REDUX
   const dispatch = useDispatch();
-  const superAdmins = useSelector((state) => state.superAdmins.superAdminsList);
+  const superAdmins = useSelector((state) => state.superAdmins.List);
+  const isLoading = useSelector((state) => state.superAdmins.isLoading);
```
Comment (cellamarematias) [inline src/Components/SuperAdmins/index.jsx:27]: I would add   `const` error= useSelector((state) => state.superAdmins.error);` So you can handle the error.

**Review by cellamarematias — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.jsx (hunk 1 above)
Comment (IvanHGo) [reply src/Components/SuperAdmins/index.jsx]: Hi! Thanks Mati, I realized that I didn't need that fetch so I changed a little the code and added only a filter. Please check it and let me know

File: src/Components/SuperAdmins/index.jsx (hunk 2 above)
Comment (IvanHGo) [reply src/Components/SuperAdmins/index.jsx:27]: Added! thanks!

**Comment by AnaMurinigo** [comment]
Test Result: Pass
GREAT work

**Review by Agustin-Chazarreta — APPROVED** [review]

**Pull request merged**